pkgs/development/node-packages/node-env.nix

```diff
@@ -446,6 +447,11 @@ let
         # Run post install hook, if provided
         runHook postInstall
       '';
+
+      meta = {
+        # default to Node.js' platforms
+        platforms = nodejs.meta.platforms;
+      } // meta;
```

Comment:
I think you accidentally added this in this PR.

Reply:
Oh, thats from https://github.com/svanderburg/node2nix/commit/58736093161f2d237c17e75a96529b018cd0ac64